dipy/reconst/shm.py

```diff
@@ -965,3 +965,80 @@ def sh_to_sf_matrix(sphere, sh_order, basis_type=None, return_inv=True, smooth=0
         return B.T, invB.T
 
     return B.T
+
+def sh_to_ap(coeffs_mtx, normal_factor=0.00001):
+    """ Calculates anisotropic power map with a given SH coefficient matrix
+
+    Notes
+    ----------
+    Calculate AP image based on a IxJxKxC SH coeffecient matrix based on the equation:
+    .. math::
```

Comment:
... based on a XxYxZxC image of SH coefficients ...
The word matrix is confusing. You basically have a vector of SH coeffs at every voxel. 

As I was reading your doc, I was a bit confused at first.